ChiModules/LinearBoltzmannSolver/lbs_02_main_execute.cc

```diff
@@ -60,19 +60,25 @@ void LinearBoltzmann::Solver::SolveGroupset(LBSGroupset& groupset,
                                      groupset.angle_agg,
                                      *sweep_chunk);
 
+  q_moments_local.assign(q_moments_local.size(), 0.0);
+
   if (groupset.iterative_method == IterativeMethod::CLASSICRICHARDSON)
   {
     ClassicRichardson(groupset, group_set_num, sweep_scheduler,
                       APPLY_MATERIAL_SOURCE |
                       APPLY_AGS_SCATTER_SOURCE |
                       APPLY_WGS_SCATTER_SOURCE |
-                      APPLY_FISSION_SOURCE);
+                      APPLY_AGS_FISSION_SOURCE |
```

Comment:
@wdarylhawkins Do you have a comment on this, from your PDT experience?

Reply:
To my knowledge, we never did this in PDT, and I can't think of any benefit off the top of my head.